Add files via upload

DimitriiKorol opened the pull request.

DimitriiKorol committed:
Add files via upload

DimitriiKorol commented:
3-й пункт пока что не сделал.

Om1X commented on 4-1/script.js:
```diff
@@ -0,0 +1,6 @@
+let text = document.querySelector('pre').textContent;
+
+text = text.replace(/'\B/ig, '"').replace(/\B'/ig, '"');
```
Почти, подумайте, как можно в одном replace. Решение с \B верное.

DimitriiKorol committed:
Add files via upload

Переделана домашка. Регулярное выражение в одну строку. Добавлен рабочий валидатор полей на основной проект, но только по кнопке отправки. Нужно для рабочей версии текущего учебного проекта на основе предыдущих доработок.

DimitriiKorol commented:
Переделал в один replace, добавил еще работающий верификатор. Переделываю старые домашки, т.к. нужна просто рабочая основа для этого учебного проекта.

DimitriiKorol committed:
Merge branch 'master' into JSplus_lession4
